packages/client-core/src/social/reducers/chat/actions.ts

```diff
@@ -22,6 +22,12 @@ import { handleCommand, isCommand } from '../../../../../common/src/utils/comman
 import { Network } from '../../../../../engine/src/networking/classes/Network'
```

Comment:
cross package imports should use @xrengine/package-name instead of relative path